tests/unittest/CMakeLists.txt

```
@@ -21,23 +21,41 @@ add_executable(unittests
     Core/raycast.cpp
     Core/resources.cpp
     Core/string.cpp
-    Core/topomesh.cpp)
+    Core/topomesh.cpp
+    Gui/keymapping.cpp)
```

```suggestion
    Gui/keymapping.cpp
    )
```
Very minor, just to ease next editions.